lxd/images.go

```diff
@@ -547,10 +547,10 @@ func getImgPostInfo(d *Daemon, r *http.Request, builddir string, project string,
 		if err != nil {
 			return nil, err
 		}
-		defer os.Remove(imageTarf.Name())
+		defer func() { _ = os.Remove(imageTarf.Name()) }()
 
 		// Parse the POST data
-		post.Seek(0, 0)
+		_, _ = post.Seek(0, 0)
```

Comment:
If seeking has failed the next reader will not work correctly, so we should return the seek error.